libsplinter/src/protocol/mod.rs

```diff
@@ -25,3 +25,14 @@ pub(crate) const ADMIN_FETCH_PROPOSALS_PROTOCOL_MIN: u32 = 1;
 
 #[cfg(feature = "proposal-read")]
 pub(crate) const ADMIN_LIST_PROPOSALS_PROTOCOL_MIN: u32 = 1;
+
+pub const SUBSCRIBE_PROTOCOL_VERSION: u32 = 1;
```

Comment:
Should this be `SCABBARD_PROTOCOL_VERSION`?

Reply:
Fixed